ddl/db_integration_test.go

```diff
@@ -93,12 +94,158 @@ func (s *testIntegrationSuite) SetUpSuite(c *C) {
 	s.tk = testkit.NewTestKit(c, s.store)
 }
 
-func (s *testIntegrationSuite) TearDownSuite(c *C) {
+func tearDownIntegrationSuiteTest(s *testIntegrationSuite, c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	r := tk.MustQuery("show tables")
+	for _, tb := range r.Rows() {
+		tableName := tb[0]
+		tk.MustExec(fmt.Sprintf("drop table %v", tableName))
+	}
+}
+
+func tearDownIntegrationSuite(s *testIntegrationSuite, c *C) {
 	s.dom.Close()
 	s.store.Close()
 }
 
-func (s *testIntegrationSuite) TestNoZeroDateMode(c *C) {
+type testIntegrationSuite1 struct {
+	testIntegrationSuite
+}
+
+func (s *testIntegrationSuite1) SetUpSuite(c *C) {
```

Comment:
We do not need to define this method for all derived type, just define it in base type `testIntegrationSuite` to avoid boilerplate code.